swiss/src/app/round/round.component.ts

```diff
@@ -277,15 +280,22 @@ export class RoundComponent implements OnInit {
   }
 
   setLargeActions() {
-    this.dialog.open(DeleteConfirmDialogComponent, { data: "všech akcí v tomto kole a vytvoření nových prázdných: 1 mise a akce pro každou aktivní jednotku" }).afterClosed().subscribe(
+    this.dialog.open(DeleteConfirmDialogComponent, { 
+      data: "všech akcí v tomto kole a vytvoření nových prázdných: 1 mise a akce pro každou aktivní jednotku, mise/jednotky navíc za neutrální rody jsou také brány v potaz"
+    }).afterClosed().subscribe(
       result => {
         if (result) {
           this.db.list("actions/" + this.roundId).remove()
           combineLatest(
             this.db.list("delegateRounds").snapshotChanges(),
             this.db.list("units").snapshotChanges(),
-            (delegateRounds, units) => {
-              return { delegateRounds: delegateRounds, units: units }
+            this.db.list("landsraad/votingRights").valueChanges(),
+            (delegateRounds, units, votingRights) => {
+              // we only need the information for the special houses which add unit or mission
+              let votingRightsFiltered = votingRights.filter((votingRight: VotingRight) => {
+                return votingRight.extraMission.toLowerCase() === "true" || votingRight.extraUnit.toLowerCase() === "true"
```

Comment:
even though we try to save boolean values, firebase somehow stores strings. I read about it and it seems other users also hit this problem as it is feature/bug of firebase. So we have to do string comparison.